db/migrate/20170110132223_add_unlimited_participants_to_events.rb

```diff
@@ -0,0 +1,5 @@
+class AddUnlimitedParticipantsToEvents < ActiveRecord::Migration
+  def change
+    add_column :events, :participants_are_unlimited, :boolean
```

Comment:
what happens with existing events? I think the default value will be nil, so its neither true nor false?
The way its implemented in the model atm will return the correct value, but maybe you could add an default value (false) anyway?